vue-app/src/components/BrightIdWidget.vue

```diff
@@ -1,21 +1,20 @@
 <template>
   <div v-if="isProjectCard" class="setup-container-project-card">
     <div class="row">
-      <div v-if="isVerified">
+      <div v-if="isLinked && isRegistered && isSponsored && isVerified">
```

Comment:
We can just check the `isRegistered` flag. If that flag is `true` that means the others are `true` as well.

If the user was added to the registry that means that the user has been verified before.